contrib/release_packages.sh

```diff
@@ -85,84 +55,48 @@ function pkg_python_checks_install_test() {
     --explain lint | grep "pythonstyle" &> /dev/null
 }
 
-PKG_PYTHON_CHECKS_CHECKER=(
-  "pantsbuild.pants.contrib.python.checks.checker"
-  "//contrib/python/src/python/pants/contrib/python/checks/checker"
-  "pkg_python_checks_checker_install_test"
-  "--universal"
-)
-function pkg_python_checks_checker_install_test() {
+function pkg_checker_install_test() {
   local version=$1
-  execute_pex \
+  execute_pex --pypi \
```

Comment:
Note: Added `--pypi` here because we _do_ want to check pypi in calls to `release.sh -t`